src/react/useLiveQuery.ts

```diff
@@ -1,35 +1,52 @@
-import { Database, DocFragment, DocTypes, IndexKeyType } from "@fireproof/core";
+import { useCallback, useEffect, useMemo, useState } from "react";
+import type { DocFragment, DocTypes, DocWithId, IndexKeyType, IndexRow, MapFn } from "@fireproof/core";
+import type { Database } from "@fireproof/core";
```

Comment:
combine rows

Reply:
seem like this can go in our linter settings